src/Microsoft.Azure.SignalR.Management/Configuration/ContextOptions.cs

```diff
@@ -12,14 +12,25 @@ internal class ContextOptions
         //Users not allowed to configure it
         internal string ProductInfo { get; set; }
 
+        /// <summary>
+        /// Gets or sets the service endpoints.
+        /// </summary>
         public ServiceEndpoint[] ServiceEndpoints { get; set; }
 
+        /// <summary>
+        /// Gets or sets the ApplicationName which will be prefixed to each hub name
```

Comment:
`which will be prefixed to each hub name` this is the implementation detail, hide it?
> Gets or sets the ApplicationName, it can be useful when you'd like to share the same Azure SignalR service for different applications with the same Hub name